### ISSUE TITLE
"List<Playlist> playlists" attribute of PlaylistManager

### ISSUE THREAD

**Issue by TylerCYan** [CSC207-UofT/course-project-tylersgroup#50]
After reviewing the code of PlaylistManager, I have found out some issues:

1. "List<Playlist> playlists" is never used any of the methods of the class
Possible solution A: delete it, if we do not care about any of the old playlists generated by all the users

Possible solution B: when we add a playlist to the user, we will also add this playlist to the "playlists" attribute. We should only do 
this if users CAN access other user's (list of) playlists and "use" their playlists. In other words, we want a "database" of playlists that were added in the past. 

Extended solution for B: when we call the removePlaylist method, it removes a playlist from the user's (list of ) playlists. If we adapt solution B, do we want to also remove this playlist from the "database"?

In short: do we want to keep track of playlists created by all the users, or each user keep track of their own playlists so we do not need the private attribute.

**Comment by JuneOh710** [comment]
Each user keeping track of only their own playlists make sense. This is database stuff so I don't think we need to worry too much about it now. If we later want to have access to all playlists, we can just start saving them all in a separate database. I think mongoDB works pretty well with Java so it should be easy.
